### PR TITLE
Fixup! Deprecation warning for sympy 1.7

### LINKED ISSUE #485: sympy 1.7 deprecation warning
Sympy 1.7 deprecated sympy.printing.ccode. We need to remove the warning

```
importing from sympy.printing.ccode has been deprecated since SymPy 1.7. 
Use Import from sympy.printing.c instead. 
See https://github.com/sympy/sympy/issues/20256 for more info.

### PR DESCRIPTION
**Pull request by cattabiani**
Fix #485

### CODE REVIEW

File: nmodl/ode.py [hunk 1]
```diff
@@ -12,7 +12,10 @@
 # import known_functions through low-level mechanism because the ccode
 # module is overwritten in sympy and contents of that submodule cannot be
 # accessed through regular imports
-known_functions = import_module('sympy.printing.ccode').known_functions_C99
+if int(sp.__version__.replace('.','')) < 17:
+    known_functions = import_module('sympy.printing.ccode').known_functions_C99
```
Comment (pramodk) [inline nmodl/ode.py:16]: versions like `1.6.1` like possible?

Comment (cattabiani) [reply nmodl/ode.py:16]: yes, I noticed when I woke up this morning. Corrected

**Comment by cattabiani** [comment]
please retest

**Review by pramodk — COMMENTED** [review]
LGTM!

one comment though : our CI install version <1.6. I assume that is still necessary for tests? (to get same output?)

https://github.com/BlueBrain/nmodl/blob/d583ae80943b576f2498d1736dc7b0712c26fa0d/azure-pipelines.yml#L41

**Comment by cattabiani** [comment]
> one comment though : our CI install version <1.6. I assume that is still necessary for tests? (to get same output?)

not at all! I fixed that with this: #441

I thought I removed that limit but, evidently, I forgot

**Comment by cattabiani** [comment]
please retest

**Review by ohm314 — APPROVED** [review]
lgtm

File: nmodl/ode.py [hunk 2]
```diff
@@ -12,11 +12,15 @@
 # import known_functions through low-level mechanism because the ccode
 # module is overwritten in sympy and contents of that submodule cannot be
 # accessed through regular imports
-known_functions = import_module('sympy.printing.ccode').known_functions_C99
+major, minor = (int(v) for v in sp.__version__.split(".")[:2])
+if major >= 1 and minor >= 7:
```
Comment (alkino) [inline nmodl/ode.py:16]: This is not correct.

2.2 will not work here.

Comment (pramodk) [reply nmodl/ode.py:16]: Stupid me! thanks Nico for pointing that!

We should convert this to integer and compare against it (`major*10+minor > 17`).

Comment (alkino) [reply nmodl/ode.py:16]: Only if minor will never be > 9

**Pull request merged**